pandas/io/clipboards.py

```diff
@@ -24,6 +26,19 @@ def read_clipboard(sep: str = r"\s+", **kwargs):  # pragma: no cover
         A string or regex delimiter. The default of '\s+' denotes
         one or more whitespace characters.
 
+    use_nullable_dtypes : bool = False
+        Whether or not to use nullable dtypes as default when reading data. If
+        set to True, nullable dtypes are used for all dtypes that have a nullable
+        implementation, even if no nulls are present.
+
+        The nullable dtype implementation can be configured by calling
+        ``pd.set_option("mode.dtype_backend", "pandas")`` to use
+        numpy-backed nullable dtypes or
+        ``pd.set_option("mode.dtype_backend", "pyarrow")`` to use
+        pyarrow-backed nullable dtypes (using ``pd.ArrowDtype``).
+
+        .. versionadded:: 2.0
+
```

Comment:
Don't we need to add that this is only supported when `engine="python"`?

Reply:
cc @mroeschke Copied this from read_csv. IIRC you added the description there? Thoughts?

Reply:
Yeah probably good to note what engines  `pd.set_option("mode.dtype_backend", "pyarrow")` is compatible with here (and read_csv too)

Reply:
Thx, added a comment for read_csv as well